packages/@okta/vuepress-site/docs/guides/event-hook-ngrok/main/index.md

```diff
@@ -153,7 +161,11 @@ To run a test of your Event Hook:
 
 1. Check your ngrok inspection interface (`http://localhost:4040`). Each call recorded by ngrok appears in the interface, from which you can review the complete call response body, header, and other details. Clicking **Replay** on the ngrok inspection interface replays the event hook call to assist in development. See [Replay the Event Hook](#replay-the-event-hook).
 
-    ![A screen shot of the ngrok inspection interface that includes details of a call.](/img/ngrok-and-event-hooks-web-interface-small.png)
+    <div class="half border">
```

Comment:
Needs to be bigger: 3/4 or full

Reply:
Same comment as above.

Reply:
Similar comments to the previous. The PR version is smaller than original. Again, that said, I'm thinking the images are not necessary.